### PR TITLE
Log cct into $artifacts_dir

### PR DESCRIPTION
**Pull request by vmoravec**
..to collect the log file from jenkins

### CODE REVIEW

**Comment by bmwiedemann** [comment]
What happens if artifacts_dir is not set or it is set, but the path does not exist?

**Comment by vmoravec** [comment]
hrm, I was expecting the $artifacts_dir always to exist, will change that.

**Comment by vmoravec** [comment]
Updated, now it's using either `$artifacts_dir` or `log/` dir which is part of the cct repo

**Comment by bmwiedemann** [comment]
as far as I can see, jenkins is the only one setting artifacts_dir and it only sets it on the mkcloud host, and we dont pass it into the crowbar VM, but rather copy files in mkcloud into $artifacts_dir

**Comment by vmoravec** [comment]
ah, would it make more sense to copy the log over to the `$artifacts_dir` at some point then?

**Comment by bmwiedemann** [comment]
yes, copying out would be better - like the screenlog.0 in mkcloud's instcrowbar step

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3601,10 +3601,11 @@ function onadmin_run_cct()
         git clone https://github.com/SUSE-Cloud/cct.git $checkout_branch
         cd cct
         bundle install
+        local log_path="${artifacts_dir:=log}/cct.log"
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: :-1: as is, because there is no $artifacts_dir here

**Comment by jdsn** [comment]
@vmoravec do you want to change this pr as proposed (copy out the logs as part of the mkcloud step, and thus moving it from qa_crowbarsetup to mkcloud), or shall we close this PR and you will create a new one?

**Comment by vmoravec** [comment]
@jdsn Let's close this PR and I will submit a new one later.

**Pull request closed without merging**